ovsdb/client.go

```diff
@@ -213,7 +213,7 @@ func (c *Client) doCallback(id int, res rpcResponse) {
 	ch, ok := c.callbacks[id]
 	if !ok {
 		// Nobody is listening to this callback.
-		panicf("OVSDB callback with ID %d has no listeners", id)
+		return
```

Comment:
So basically, we just ignore the message unless there is a callback to the corresponding ID.  Yeah, that's better than just crash because of the malicious/unexpected RPC reply.

LGTM